vendor/github.com/clarketm/json/decode.go

```diff
@@ -85,14 +85,13 @@ import (
 //
 // The JSON null value unmarshals into an interface, map, pointer, or slice
 // by setting that Go value to nil. Because null is often used in JSON to mean
-// ``not present,'' unmarshaling a JSON null into any other Go type has no effect
+// “not present,” unmarshaling a JSON null into any other Go type has no effect
```

Comment:
This is weird we should prob not format the 'vendor' folder.

This being true for all files under the "vendor" folder.